### PR TITLE
Changes for Exporter completion page.

### PR DESCRIPTION
**Pull request by AtosSK**

### CODE REVIEW

File: EPRN.PRNS.API/appsettings.json [hunk 1]
```diff
@@ -5,5 +5,10 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "AppSettings": {
+    "DeductionAmount_Exporter": 0.15,
+    "DeductionAmount_Reprocessor": 0.16,
+    "DeductionAmount_ExporterAndReprocessor": 0.17
+  }
```
Comment (NikPeters-Atos) [inline EPRN.PRNS.API/appsettings.json:10]: Why are there app settings in PRN api, yet no other changes? Are these required?

File: EPRN.Portal/Controllers/WasteController.cs [hunk 2]
```diff
@@ -150,10 +150,14 @@ public async Task<IActionResult> SubTypes(WasteSubTypesViewModel wasteSubTypesVi
...
-        public async Task<IActionResult> GetWasteRecordStatus(int journeyId)
+        public async Task<IActionResult> GetWasteRecordStatus(int? id)
         {
-            var result = await _wasteService.GetWasteRecordStatus(journeyId);
+            if (id == null)
+                return NotFound();
+
+            var result = await _wasteService.GetWasteRecordStatus(id.Value);
+
+            result.WasteRecordReferenceNumber = "WR00012";
```
Comment (NikPeters-Atos) [inline EPRN.Portal/Controllers/WasteController.cs]: Even though we don't know how this is generated at the moment, it should be set in the service layer, not the Controller layer.

File: EPRN.Portal/Services/HomeServices/BaseHomeService.cs [hunk 3]
```diff
@@ -14,14 +14,14 @@ public BaseHomeService(IOptions<AppConfigSettings> configSettings)
...
 
-            if (configSettings.Value.DeductionAmount_Reprocessor == null)
-                throw new ArgumentNullException(nameof(configSettings.Value.DeductionAmount_Reprocessor));
+            //if (configSettings.Value.DeductionAmount_Reprocessor == null)
+            //    throw new ArgumentNullException(nameof(configSettings.Value.DeductionAmount_Reprocessor));
 
-            if (configSettings.Value.DeductionAmount_ExporterAndReprocessor == null)
-                throw new ArgumentNullException(nameof(configSettings.Value.DeductionAmount_ExporterAndReprocessor));
+            //if (configSettings.Value.DeductionAmount_ExporterAndReprocessor == null)
+            //    throw new ArgumentNullException(nameof(configSettings.Value.DeductionAmount_ExporterAndReprocessor));
```
Comment (NikPeters-Atos) [inline EPRN.Portal/Services/HomeServices/BaseHomeService.cs]: We shouldn't be checking in commented out code. Review and ensure the correct code is being run

**Review by NikPeters-Atos — COMMENTED** [review]
Should there be any unit tests as well? Do all unit tests still run?

File: EPRN.PRNS.API/appsettings.json (hunk 1 above)
Comment (AtosSK) [reply EPRN.PRNS.API/appsettings.json:10]: Removed.

File: EPRN.Portal/Controllers/WasteController.cs (hunk 2 above)
Comment (AtosSK) [reply EPRN.Portal/Controllers/WasteController.cs]: Done.

File: EPRN.Portal/Services/HomeServices/BaseHomeService.cs (hunk 3 above)
Comment (AtosSK) [reply EPRN.Portal/Services/HomeServices/BaseHomeService.cs]: Done.

File: EPRN.Portal/Services/HomeServices/BaseHomeService.cs [hunk 4]
```diff
@@ -11,6 +11,9 @@ public abstract class BaseHomeService : IHomeService
 
         public BaseHomeService(IOptions<AppConfigSettings> configSettings)
         {
+            if (configSettings == null)
+                return;
+
```
Comment (NikPeters-Atos) [inline EPRN.Portal/Services/HomeServices/BaseHomeService.cs]: Don't need these lines. The issue over the loading of the config has been resolved.

File: EPRN.Waste.API/Services/JourneyService.cs [hunk 5]
```diff
@@ -95,7 +95,7 @@ public async Task<WasteRecordStatusDto> GetWasteRecordStatus(int journeyId)
             {
                 JourneyId = journey.Id,
                 WasteBalance = journey.Quantity ?? 0,
-                WasteRecordReferenceNumber = journey.ReferenceNumber,
+                WasteRecordReferenceNumber = journey.ReferenceNumber ?? "WR00012",
```
Comment (NikPeters-Atos) [inline EPRN.Waste.API/Services/JourneyService.cs:99]: Put a comment or a TODO comment explaining why this is hard coded, and that it'll be removed in time

File: EPRN.Portal/Services/HomeServices/BaseHomeService.cs (hunk 4 above)
Comment (AtosSK) [reply EPRN.Portal/Services/HomeServices/BaseHomeService.cs]: Done.

File: EPRN.Waste.API/Services/JourneyService.cs (hunk 5 above)
Comment (AtosSK) [reply EPRN.Waste.API/Services/JourneyService.cs:99]: Done.

**Review by NikPeters-Atos — APPROVED** [review]

**Pull request merged**